policyengine_us/parameters/gov/states/nm/tax/income/exemptions/low_and_middle_income/income_threshold.yaml

```diff
@@ -0,0 +1,24 @@
+description: New Mexico provides the low- and middle income exemption for filers below this agi threshold.
+metadata:
+  label: New Mexico low and middle income exemption income threshold
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 New Mexico personal income tax form packet
+    href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf
+  - title: 
```

Comment:
add 2021 forms

Reply:
found the file of the [2021 New Mexico personal income tax form packet](https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/608063af-0c4c-4f28-ba42-cf40e04557ca/PITbook2021.pdf)